src/content/containers/Home.js

```diff
@@ -141,13 +166,14 @@ class Home extends PureComponent {
           <h2>Getting started</h2>
           <p>
             Install the Gecko Profiler Add-on to start recording a performance profile in
-            Firefox, then analyze it and share it with perf.html, or drag a saved profile
-            here to load it in perf.html.
+            Firefox, then analyze it and share it with perf.html.
           </p>
           <InstallButton name='Gecko Profiler' className='homeSectionInstallButton' xpiURL={ADDON_URL}>
             <span className='homeSectionPlus'>+</span>
             Install add-on
           </InstallButton>
+          <p>You can also upload a saved profile by either dragging and dropping it here or selecting it using the button below.</p>
```

Comment:
And now I realize that we don't really  `upload` as everything stays local. How about replacing `upload a saved profile` by `analyze a local profile` ?